app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -218,7 +218,7 @@ private String getOrderingStatusValidRequest() {
             "collectionStatus",
             "DRAFT",
             "examOrderDateTime",
-            "2022-12-08T17:45:61Z",
+            "2022-12-08T17:45:59Z",
```

Comment:
ISO 1806 dates don't permit 61 seconds (no date standard does)
This was never valid we just didn't use it. It's now valid.